src/20240617_AaveV3Avalanche_UpdatePoRExecutorV3Robot/UpdatePoRExecutorV3Robot.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Update PoR Executor V3 Robot"
+author: "BGD Labs (@bgdlabs)"
+discussions: ""
+---
+
+## Simple Summary
+
+Proposal to give new Proof of Reserve Executor V3 risk admin role on Avalanche and register the new robot.
+
+## Motivation
+
+With the release of Aave V3.1, it is no longer necessary to set the asset's LTV to zero before freezing during the execution of an emergency action on a Proof of Reserve alert. Moreover setting LTV to zero will break the setting of LTV back to normal value on unfreeze.
```

Comment:
```suggestion
With the release of Aave V3.1, it is no longer necessary to set the asset's LTV to zero before freezing during the execution of an emergency action on a Proof of Reserve alert, as the protocol does it both actions in batch. Moreover setting LTV to zero that way would break the "rollback" mechanism (pendingLtv) of LTV back to normal value on unfreeze.
```